openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -919,10 +919,10 @@ This specification defines the following proof types:
 
 There are two ways to convey key attestations (as defined in (#keyattestation)) of the cryptographic key material during Credential issuance:
 
-- The Wallet uses the `jwt` proof type in the Credential Request to create a proof of possession of the key and adds the key attestation in the JOSE header.
-- The Wallet uses the `attestation` proof type in the Credential Request with the key attestation without a proof of possession of the key itself.
+- The Wallet uses the `jwt` proof type in the Credential Request to create a proof of possession for one of the attested keys and adds the key attestation in the JOSE header.
```

Comment:
```suggestion
- The Wallet uses the `jwt` proof type in the Credential Request to create a proof of possession for one of the attested keys and adds the key attestation with all of the attested keys in the JOSE header.
```

Reply:
I agree with @jogu, that suggestion is clearer than this proposed language.